prow/cluster/BUILD

```diff
@@ -0,0 +1,28 @@
+load("@io_bazel_rules_k8s//k8s:object.bzl", "k8s_object")
+
+# Usage:
+#   bazel run //prow/cluster:foo.apply
+
+# TODO(fejta): consider k8s_deploy alias
+# https://github.com/bazelbuild/rules_k8s#aliasing-eg-k8s_deploy
+
+k8s_object(
+    name = "branchprotector_cronjob",
+    cluster = "{STABLE_PROW_CLUSTER}",  # Run make :get-cluster-credentials
+    kind = "Job",
```

Comment:
CronJob?

Reply:
This is another reason I'm biasing away from it :-/

Reply:
Done